Toolkit/ArcGISToolkit.xcodeproj/project.pbxproj

```diff
@@ -31,6 +29,8 @@
 		E46EF2C7236CA16800A8C50B /* BookmarksTableViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = E46EF2C5236CA16800A8C50B /* BookmarksTableViewController.swift */; };
 		E48405731E9BE7B700927208 /* LegendViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = E48405721E9BE7B700927208 /* LegendViewController.swift */; };
 		E484057A1E9C262D00927208 /* Legend.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = E48405791E9C262D00927208 /* Legend.storyboard */; };
+		E4E7E57A255C5193002458B9 /* ArcGIS.xcframework in Frameworks */ = {isa = PBXBuildFile; fileRef = E4E7E579255C5193002458B9 /* ArcGIS.xcframework */; };
+		E4E7E57B255C5193002458B9 /* ArcGIS.xcframework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = E4E7E579255C5193002458B9 /* ArcGIS.xcframework */; settings = {ATTRIBUTES = (CodeSignOnCopy, ); }; };
```

Comment:
The `ArcGIS` framework should not be embedded by the `ArcGISToolkit` framework. Please fix.
![image](https://user-images.githubusercontent.com/2257493/98862656-20a06980-241c-11eb-8f47-e2afc494075a.png)

Reply:
Excellent, thanks.